chessllm/chess_utils.py

```diff
@@ -1,7 +1,8 @@
 import io
 import re
 from datetime import datetime
-
+# import os
+# os.add_dll_directory(r"C:\Program Files\GTK3-Runtime Win64\bin") Uncomment this line if running on windows and GTK3 is installed
```

Comment:
Hmm, there must be a better way than commenting this in and out.

Reply:
Agreed, maybe I can check which os we are running on and add the import . Also will add the GTK installation step in readme.md

Reply:
FIxed comment by adding import only if platform is Windows